### PR TITLE
docs - 44 updates

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Adds note about new bulk filter modal.
- Adds note about variables in text cards.

### CODE REVIEW

File: docs/users-guide/07-dashboards.md [hunk 1]
```diff
@@ -65,6 +65,30 @@ Click the **eye** icon to see what your formatted Markdown will look like when y
...
+
+You can also make text options by wrapping the text in double brackets, `[[` and `]]`:
+
+```
+{% raw %}
+# Orders [[from {{state}}]
+{% endraw %}
+```
+
+In this case, the phrase `from {{state}}` would only display if someone selected a value (or values) in the filter widget.
```
Comment (jeff-bruemmer) [inline docs/users-guide/07-dashboards.md:88]:
```suggestion
In this case, the phrase `{% raw %}from {{state}}{%endraw%}` would only display if someone selected a value (or values) in the filter widget.
```

File: docs/users-guide/07-dashboards.md [hunk 2]
```diff
@@ -65,6 +65,30 @@ Click the **eye** icon to see what your formatted Markdown will look like when y
 
 To learn more, see [Fun with Markdown in your dashboards](https://www.metabase.com/blog/markdown-in-dashboards/index.html).
 
+### Including variables in text cards
+
+You can include a variable in a text card, then wire that variable up to a dashboard filter. All you need to do to create a variable is to wrap a word in double braces, `{{` and `}} (the variable can't contain any spaces). For example, you could add a text card with the following text:
```
Comment (nllho) [inline docs/users-guide/07-dashboards.md:70]:
```suggestion
You can include a variable in a text card, then wire that variable up to a dashboard filter. All you need to do to create a variable is to wrap a word in double braces, `{{` and `}}` (the variable can't contain any spaces). For example, you could add a text card with the following text:
```

File: docs/users-guide/04-asking-questions.md [hunk 3]
```diff
@@ -75,6 +75,14 @@ Once you're happy with your filter, click **Add filter**, and visualize your res
 
 If you want to edit your filter, just click the little purple filter at the top of the screen. If you click on the X, you'll remove your filter. You can add as many filters as you need.
 
+## Filter modal
+
+When viewing a table or chart, clicking on the **Filter** will bring up the filter modal:
+
+![Bulk filter modal](./images/asking-questions/bulk-filter-modal.png)
+
+Here you can add multiple filters to your question in one go. Filter options will differ depending on the field type. Any tables linked by foreign keys will be displayed in the left tab of the modal. When you're done adding filters, hit **Apply filters** to rerun the query and update its results. To remove all the filters you've applied, click on **Clear all filters** in the bottom right of the filter modal. Any filters you apply here will show up in the notebook editor, and vice versa.
```
Comment (nllho) [inline docs/users-guide/04-asking-questions.md:84]: link for "field type":

https://www.metabase.com/docs/latest/users-guide/field-types.html

**Review by nllho — APPROVED** [review]

File: docs/users-guide/04-asking-questions.md (hunk 3 above)
Comment (jeff-bruemmer) [inline docs/users-guide/04-asking-questions.md:84]:
```suggestion
Here you can add multiple filters to your question in one go. Filter options will differ depending on the [field type](./field-types.md). Any tables linked by foreign keys will be displayed in the left tab of the modal. When you're done adding filters, hit **Apply filters** to rerun the query and update its results. To remove all the filters you've applied, click on **Clear all filters** in the bottom right of the filter modal. Any filters you apply here will show up in the notebook editor, and vice versa.
```

**Pull request merged**